sdk/keyvault/test-resources.json

```diff
@@ -65,6 +65,13 @@
       "metadata": {
         "description": "The base time to add 2 hours to for SAS token expiration. The default is the current time."
       }
+    },
+    "attestationBaseName": {
```

Comment:
Probably better to give the whole URL. I called mine `attestationUrl` and defaulted it to yours. If we get a real attestation service, it likely won't be on azurewebsites.net.

Reply:
My rationale for using a basename and outputting the entire URL was I imagined when we deploy this dynamically that you'd input the base name, we'd deploy a resource (app service or static resource) and then output the URL so this matches more closely to what I imagined the end-result would be as far as input and output to the template. 

With that said, I think either is fine - so I am happy to change mine. What do you think?

Reply:
In different clouds even the domain name will often change. Better to use full host names across the board. For example, the test-resources.json that everyone copied from me outputs the full KV and MHSM hostnames.

That said, to keep variables grouped I'm going with AZURE_KEYVAULT_ATTESTATION_URL for a variable name. Given everyone has copied from me, just a thought to unify since it might happen again. I've also set the retention days to 7 (minimum). Too many KVs blocking me from names I want to use. Most expire this week, fortunately.